application/src/main/java/org/opentripplanner/osm/model/OsmWay.java

```diff
@@ -130,6 +134,19 @@ public boolean isEscalator() {
     return (isTag("highway", "steps") && isOneOfTags("conveying", ESCALATOR_CONVEYING_TAGS));
   }
 
+  public Optional<Duration> getDuration() {
```

Comment:
Please add javadoc about the expected format.

Also, if you want to fix them, consider adding an issue. Check https://github.com/opentripplanner/OpenTripPlanner/blob/79221b66fc7f6a17b6de5176cbde1c6e52881603/application/src/main/java/org/opentripplanner/osm/model/OsmWithTags.java#L208-L222 for an example.